2-mandatory/4-stocks.js

```diff
@@ -34,7 +34,17 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
 function getAveragePrices(closingPricesForAllStocks) {
-    // TODO
+
+    let agv = 0;
```

Comment:
think there's a typo here

Reply:
It was extra. I changed my mind but I forgot to delete.